Lib/asyncio/locks.py

```diff
@@ -57,7 +57,7 @@ def __exit__(self, *args):
 
     @coroutine
     def __iter__(self):
-        # This is not a coroutine.  It is meant to enable the idiom:
+        # This is not a coroutine function.  It is meant to enable the idiom:
```

Comment:
"This is not a coroutine function." -> just remove this sentence, it doesn't make a lot of sense anyways.